src/ipc-handlers.ts

```diff
@@ -208,7 +211,7 @@ export async function startServer(
 
 	const parentWindow = BrowserWindow.fromWebContents( event.sender );
 	await server.start();
-	if ( parentWindow ) {
+	if ( parentWindow && ! parentWindow.isDestroyed() ) {
```

Comment:
Is there a particular reason why this guard does not include `! event.sender.isDestroyed()` while all others do?

Reply:
Missed! Thanks! :)